modules/test/playwright/tests/osb-faro-web/utils/definitions.ts

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
@ViniLopes20 can we rename this to `eventDefinitions.ts`? I think it would be more self-explained :wink:

Reply:
Done